birdy/client/base.py

```diff
@@ -196,11 +196,12 @@ def _get_process_description(self, processes=None, xml=None):
         all_wps_processes = [p.identifier for p in self._wps.processes]
 
         if processes is None:
-            if owslib.__version__ > "0.17.0":
+            try:
                 # Get the description for all processes in one request.
-                ps = self._wps.describeprocess("all", xml=xml)
+                ps = self._wps.describeprocess("ALL", xml=xml)
                 return OrderedDict((p.identifier, p) for p in ps)
-            else:
+
+            except ServiceException as e:
```

Comment:
owslib is raising a ValueError ... our tests are failing.

Reply:
I think it's because I switched from "all" to "ALL", per the WPS specs. Reverted to "all" and catched the ValueError.